recipes/pyweed/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pyweed" %}
+{% set version = "0.5.2.dev0" %}
+{% set sha256 = "3e1e8e35cba3f09a52a540b4351042d9e62a98bc430d7f70315df621fb2177f1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - obspy
+    - pyqt 4.11*
+    - qtconsole
+    - basemap
+    - pyproj
+    - pillow
+    - mock  # [py2k]
+
+test:
```

Comment:
Typically we ask that you test all command line tools using:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

but I'm not sure what the proper approach is given that this launches a GUI.

Reply:
I changed this to execute the unit tests